components/list/list-header.js

```diff
@@ -60,6 +89,9 @@ class ListHeader extends RtlMixin(LocalizeCoreElement(LitElement)) {
 			:host([padding-type="slim"]) .d2l-list-header-container {
 				min-height: 36px;
 			}
+			.d2l-list-header-extend-separator {
```

Comment:
This is just adding support for the extend-separators case that was missed when `d2l-list-header` was first introduced.